sasdata/fair_database/data/tests.py

```diff
@@ -0,0 +1,123 @@
+import os
+import shutil
+
+from django.conf import settings
+from django.test import TestCase
+from django.contrib.auth.models import User
+from rest_framework.test import APIClient, APITestCase
+from rest_framework import status
+
+from .models import Data
+
+def find(filename):
+    return os.path.join(os.path.dirname(__file__), "../../example_data/1d_data", filename)
+
+class TestLists(TestCase):
```

Comment:
Other suggested tests:

- Modify data using PUT (public data while authenticated and not, private data while authenticated as data owner, authenticated as other user, and unauthenticated)
- DELETE public and private data (all user cases listed in last bullet) - Should public data be deletable? By the administrator, yes. By a random user? That's a question I'll think about...